helm/g2p-sandbox/values.yaml

```diff
@@ -377,6 +377,38 @@ ph-ee-engine:
         batch_transaction: "/api/v1/batch/transactions"
     identity_account_mapper:
       hostname: "https://identity-mapper.sandbox.mifos.io"
+    extraEnv:
+      - name: "BUDGET_ACCOUNT_REGISTERINGINSTITUTIONSID_0"
```

Comment:
is the original design for adding these values to the Helm chart approved? I feel the better place for these configurations are config map. (ph-ee-config) 
They are more business level than system level to be added to deployment configuration;

Reply:
A OevOps user may not be able to understand and make decisions on these values

Reply:
This PR was created for testing purpose and to understand how the extraEnv part works.It should not be merged as it has all the values hardcoded which should be added in the config map.